### ISSUE TITLE
[FEATURE] Display current level in towns

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#298]
Just realised this, but would it not be reasonable to display the current level in a town, like in dungeons? I may be able to work this easily with existing code.

There are up to 6 levels in towns now, so, could be helpful for some people.

Just a thought.

**Comment by cambragol** [comment]
Here is a version working:

However, the naming convention of L for above ground and B for lower levels messes with the dungeons. They seem off now. To implement would mean having to change dungeon levels from L to B. So B1, B2, B3 etc.

Hmm.

![levels](https://user-images.githubusercontent.com/5019739/155881145-ba5f115a-6270-4ec0-a42e-a9fc4eb121ce.gif)

**Comment by MrKnitster** [comment]
Also to keep in mind--is there ever a need to know the moon phases while in a town? If so, of course, they would not be visible. I don't think there is a need currently, but something to think about.

**Comment by cambragol** [comment]
No, there isn't. Whch is why the moons don't update in town. It is also the 'scale' thing, where time in town is at a different physical and temporal scale. Which is now reflected in how food/poison/sleep are handled in town: at 1/10 the rate of the outdoors.

My question for above is: Is it weird to have dungeons using L1-L8, while the above town portions of towns use the same labelling (up to L4) but then use B1 - B2 for basements?

**Comment by MrKnitster** [comment]
I think it's fine to have all levels be L rather than B for basements, unless it is easily possible. Some towns, of course, have both, which _would_ be kinda neat

**Comment by cambragol** [comment]
But how would numbering work for towns then? The levels below ground would need a negative, which doesn't fit. So it has to be B, as in the gif above, otherwise it won't fit or make sense. 

A system where town level are B2,B1,L1,L2,L3,L4 and dungeons are L1,L2,...L8, is the easiest implementation and requires only a small change to code.

Tweaking it so Dungeons are B1,B2,...B8 is slightly harder, but just makes a little more sense to me. But Maybe not. 
A 'Dungeon' is a different entity than a town, so perhaps deserves the L label for its levels. That is the tradition I believe. So why change it? I can't think of any instances of dungeons with B labeling, but plenty with L labeling. Gauntlet comes to mind immediately.

**Comment by cambragol** [comment]
Looking at the gif above, there is still an issue. At least in that, one, location. Moving from L1 of Castle Britannia, to the L1 of Hythloth has no change. They are both level one. So aside from that case, everything works. But this case makes things weird.

**Comment by MrKnitster** [comment]
When you move from level to level in dungeons, doesn't the game produce the message "DOWN TO LEVEL 3" or something similar? All that would need to be changed as well. Would it say "DOWN TO BASEMENT 3"? That would be weird.

**Comment by cambragol** [comment]
No, that would be unaffected. I think the best scenario, if I was to make this change, is as displayed in the gif above. Naming for Dungeon levels is unchanged. Towns use B2,B1,F1,F2,F3, and F4. The descent down the ladder in Castle Britannia would just have to be shrugged off.
